index.html

```diff
@@ -4,6 +4,13 @@
 <head>
   <meta charset="utf-8">
   <title>HTML Academy: Седона</title>
+
+  <!-- Фавиконки  -->
+
+  <link rel="icon" href="favicon.ico">
+  <link rel="icon" href="favicon/icon.svg" type="image/svg+xml">
```

Comment:
```suggestion
  <link rel="icon" href="img/favicon/icon.svg" type="image/svg+xml">
```

Так как папка **favicon** находится в папке **img**, а не в корне проекта.